ansible/templates/whisk.properties.j2

```diff
@@ -53,3 +53,5 @@ use.cli.download=false
 nginx.conf.dir={{ nginx_conf_dir }}
 testing.auth={{ openwhisk_home }}/config/keys/auth.guest
 vcap.services.file=
+limit.entity.actioncode=48m
+limit.entity.parameters=1m
```

Comment:
I would prefer writing the limits directly into the code.
Then it is much easier to access them everywhere.
Another advantage is, that it can be directly generetated as ByteSize object. So it doesn't have to be parsed.